misk-hibernate/src/main/kotlin/misk/vitess/StartVitessService.kt

```diff
@@ -349,9 +343,41 @@ class StartVitessService(
 
   fun cluster() = cluster!!.cluster
 
+  fun shouldRunVitess() =  config.type != DataSourceType.VITESS || environment == TESTING || environment == DEVELOPMENT
+
   override fun shutDown() {
   }
 
+  /**
+   * Shut down the cached clusters on JVM exit.
+   */
+  init {
```

Comment:
moved from Companion.Init to StartVitessService.Init so we can check the Environment. Shouldn't impact anything, it still checks the `imagedPulled` var